OMX-27.ino

```diff
@@ -163,10 +164,10 @@ void advanceClock(Micros advance) {
 void resetClocks(){
 	// BPM tempo to step_delay calculation
 	clockInterval = 60000000/(PPQ * clockbpm); // clock interval is in microseconds
-	step_micros = clockInterval * 6; 			// 16th note step in microseconds
+	step_micros = clockInterval * (PPQ/4);			// 16th note step in microseconds
```

Comment:
@okyeron thoughts on this?

Reply:
if PPQ is 24 the those two lines are equivalent. If PPQ is 48 then the interval changes.